editor/block-mover/test/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { BlockMover } from '../';
+
+describe( 'BlockMover', () => {
+	describe( 'basic rendering', () => {
+		it( 'should render two IconButton components with the following props', () => {
+			const blockMover = shallow( <BlockMover /> );
+			expect( blockMover.hasClass( 'editor-block-mover' ) );
+
+			const moveUp = blockMover.childAt( 0 );
+			const moveDown = blockMover.childAt( 1 );
+			expect( moveUp.type().name ).to.equal( 'IconButton' );
+			expect( moveDown.type().name ).to.equal( 'IconButton' );
+			expect( moveUp.props() ).to.include( {
+				className: 'editor-block-mover__control',
+				onClick: undefined,
```

Comment:
The behavior of `includes` and `undefined` here is such that even if the `IconButton` _did_ assign an `onClick` handler, the test would still pass. You can see this with the following diff applied:

```diff
diff --git a/editor/block-mover/index.js b/editor/block-mover/index.js
index f040d239..cba1a09c 100644
--- a/editor/block-mover/index.js
+++ b/editor/block-mover/index.js
@@ -25,7 +25,7 @@ function BlockMover( { onMoveUp, onMoveDown, isFirst, isLast } ) {
                <div className="editor-block-mover">
                        <IconButton
                                className="editor-block-mover__control"
-                               onClick={ isFirst ? null : onMoveUp }
+                               onClick={ isFirst ? function() {} : onMoveUp }
                                icon="arrow-up-alt2"
                                aria-disabled={ isFirst }
                        />
```